tests/serialization/triangulation_01.output

```diff
@@ -2,33 +2,33 @@
 DEAL::0 0 0 0 0 3 0 2 1 0
 0 1 0 0 1 0 0 0 2 0 0 0 -1 -1 -1 -1 1 0 0 1 0 0 1 0 -1 0 0 0 0 0 1 0 0 0 2 0 1 1 0 0 0 0 0 0 1 1 1 0 0 0 1 0 0 0 0 1 0 4294967295 0 0 1 0 0 1 0 0 0 0 0 2
 1 2 0 0 0 2 0 0 4 0 -1 -1 1 1 1 0 -1 -1 2 0 0 0 2 0 0 0 1 0 0 0 2 0 2 0 2 2 2 1 2 0 0 2 0 0 2 1 1 2 0 0 2 0 0 0 2 0 4294967295 4294967295 1 0 1 2 0 0 0 0 2
-2 4 0 1 0 0 0 4 0 0 0 0 8 0 -1 -1 2 1 2 0 2 2 2 1 2 3 2 2 -1 -1 4 0 1 0 0 0 4 0 4 0 0 0 2 0 0 1 0 4 0 2 0 3 2 3 2 2 2 4 2 4 1 4 0 -1 -1 -1 -1 0 0 4 1 1 1 1 4 1 0 0 0 4 0 2 0 0 0 4 0 4294967295 4294967295 4294967295 4294967295 3 0 1 4 0 3 0 0 0 2 -1 0 0 5 0 0 0 0 0 1 0 0 0.00000000000000000e+00 1 1.00000000000000000e+00 1 5.00000000000000000e-01 1 2.50000000000000000e-01 1 7.50000000000000000e-01 5 1 1 1 1 1 0 0 0 3 7 3 0 1 2 4 4 3 0 0 0 4 0 23 1 0
-3 2 0 0 0 0 0 1 1 25 1 0
+2 4 0 1 0 0 0 4 0 0 0 0 8 0 -1 -1 2 1 2 0 2 2 2 1 2 3 2 2 -1 -1 4 0 1 0 0 0 4 0 4 0 0 0 2 0 0 1 0 4 0 2 0 3 2 3 2 2 2 4 2 4 1 4 0 -1 -1 -1 -1 0 0 4 1 1 1 1 4 1 0 0 0 4 0 2 0 0 0 4 0 4294967295 4294967295 4294967295 4294967295 3 0 1 4 0 3 0 0 0 2 0 0 -1 0 0 5 0 0 0 0 0 1 0 0 0.00000000000000000e+00 1 1.00000000000000000e+00 1 5.00000000000000000e-01 1 2.50000000000000000e-01 1 7.50000000000000000e-01 5 1 1 1 1 1 0 0 0 3 7 3 0 1 2 4 4 3 0 0 0 4 0 0 0 25 1 0
+3 2 0 0 0 0 0 1 1 0 0 28 1 0
```

Comment:
boost encodes raw pointers differently than `std::unique_ptr` objects. I've never understood how it does that in detail, but it needs to encode both the type and the members of the class in the stream of numbers.